spec/src/main/asciidoc/cdi-spec.asciidoc

```diff
@@ -88,7 +88,4 @@ include::core/packagingdeployment_full.asciidoc[]
 
 include::javase/javase.asciidoc[]
 
-[[part_3]]
-= Part III - CDI in Jakarta EE
 
-include::javaee/javaeeintegration.asciidoc[]
```

Comment:
Shouldn't we also remove this file and all the other EE related files it points to?

Reply:
Yes, for sure. I will add that to this PR once it is clear the contents and history have been migrated over to the platform repo.

Reply:
Ok, EE files have been removed